### ISSUE TITLE
radicale_infcloud out of date

### ISSUE THREAD

**Issue by MalteT** [NixOS/nixpkgs#150983]
The current version is incompatible with Radicale 3.0.x
###### Checklist

- [x] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name: `nix search radicale_infcloud` (What is expected here?)

current version: `2017-07-27` (`972757bf4c6be8b966ee063e3741ced29ba8169f`)
desired version: `2020-08-07` (`40f88a5dc0003120d62533ea635f0d94d77f1aca`)

###### Notify maintainers

maintainers: @aneeshusa 

###### Note for maintainers

Please tag this issue in your PR.

**Comment by MalteT** [comment]
They've added a dependency on Radicale which I just removed in my overlay, but there might be a better way:
```diff
diff --git a/setup.py b/setup.py
index 0e1bf2a..c3bae30 100755
--- a/setup.py
+++ b/setup.py
@@ -23,4 +23,4 @@ setup(
     platforms="Any",
     packages=["radicale_infcloud"],
     package_data={"radicale_infcloud": web_data},
-    install_requires=["radicale>=3.0.0"])
+    install_requires=[])
```

**Comment by JamieMagee** [comment]
Resolved by #191333
